pkg/maps/lbmap/ipv6.go

```diff
@@ -32,16 +32,35 @@ const (
 
 	// SockRevNat6MapSize is the maximum number of entries in the BPF map.
 	SockRevNat6MapSize = 256 * 1024
+
+	// Service6MapV2Name is the name of the IPv6 LB Services v2 BPF map.
+	Service6MapV2Name = "cilium_lb6_services_v2"
+	// Backend6MapName is the name of the IPv6 LB backends BPF map.
+	Backend6MapName = "cilium_lb6_backends"
+	// RevNat6MapName is the name of the IPv6 LB reverse NAT BPF map.
+	RevNat6MapName = "cilium_lb6_reverse_nat"
 )
 
 var (
 	// MaxSockRevNat6MapEntries is the maximum number of entries in the BPF map.
 	// It is set by InitMapInfo(), but unit tests use the initial value below.
 	MaxSockRevNat6MapEntries = SockRevNat6MapSize
+
+	// The following BPF maps are initialized in InitializeIPv6().
+
+	// Service6MapV2 is the IPv6 LB Services v2 BPF map.
+	Service6MapV2 *bpf.Map
+	// Backend6Map is the IPv6 LB backends BPF map.
+	Backend6Map *bpf.Map
+	// RevNat6Map is the IPv6 LB reverse NAT BPF map.
+	RevNat6Map *bpf.Map
```

Comment:
We need to initialize `SourceRangeMap{4,6}`, `AffinityMatchMap`, `Affinity{4,6}Map` too, as their size is `MaxEntries`.

Reply:
> For Affinity{4,6}Map, are they still in use?

Yeah, they are, but only from the datapath. Nevertheless, we create them in the agent so that any map size mismatch could be detected, and then the maps could be recreated.